ArjunaJTA/jee_transactional_app/pom.xml

```diff
@@ -13,6 +13,13 @@
 	Fifth Floor, Boston, MA 02110-1301, USA. (C) 2009 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jboss.narayana.quickstart.jta</groupId>
+        <artifactId>narayana-quickstarts-jta</artifactId>
+        <version>5.10.5.Final-SNAPSHOT</version>
+    </parent>
+
     <groupId>org.jboss.narayana.quickstart.jta</groupId>
     <artifactId>jee_transactional_app</artifactId>
     <version>5.10.5.Final-SNAPSHOT</version>
```

Comment:
please, do not use the `<version>` twice in the `pom.xml`. Here you define the artifact `<artifactId>jee_transactional_app` which inherits the version from the parent.

It means the version for the artifact needs to be removed from all(*!*) the `pom.xml` where you added the `<parent>`.

Reply:
okay, I will remove it from all the POM files